CODE_OF_CONDUCT.md

```diff
@@ -35,8 +42,161 @@ Sage can more effectively collaborate with others if they follow this code.
+the content of complaints as confidential (unless the reporter agrees
+otherwise).
+
+In case of severe and obvious breaches, e.g., personal threat or
+violent, sexist or racist language, we will immediately disconnect the
+originator from Sage communication channels; please see the manual
+for details.
+
+In cases not involving clear severe and obvious breaches of this code
+of conduct, the process for acting on any received code of conduct
+violation report will be:
+
+- acknowledgement that the report has been received
+- discussion within the committee
+- discussion with and/or feedback provided to the reportee
+- mediation (if feedback didn’t help, and only if both reporter and
+  reportee agree to this)
+- enforcement via transparent decision by the sasge-abuse committee
+
+The committee will respond to any report as soon as possible, and our
+goal will be to respond within 72 hours.
+
+Potential consequences for violating the Sage code of conduct include:
+
+- Nothing (if we determine that no violation occurred)
+- Private feedback or reprimand to the individual(s) involved
+- Warning the person to cease their behavior and that any further
+  reports will result in sanctions
+- A public announcement that an incident occurred
+- Mediation (only if both reporter and reportee agree)
+- An imposed vacation (e.g. asking someone to "take a week off" from a mailing list)
+- A permanent or temporary ban from some or all Sage spaces (mailing
+  lists, GitHub repos, in-person events, etc.)
+- Assistance to the reporter with a report to other bodies, for
+  example, institutional offices or appropriate law enforcement
+  agencies
+- Publishing an account of the harassment and calling for the
+  resignation of the alleged harasser from their responsibilities
+  (usually pursued by people without formal authority: may be called
+  for if the person is the event leader, or refuses to stand aside
+  from the conflict of interest, or similar)
+- Any other response that the Sage Code of Conduct Committee deems necessary and
+  appropriate to the situation
+
+## Policies and procedures of the project's pertinent platforms ##
+
+[GitHub Community Guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines) apply
+- [Report abuse to GitHub](https://github.com/contact/report-abuse),
+  [Block a user from your personal account](https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account)
+- [Manage disruptive comments](https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-disruptive-comments),
+  [Lock conversations](https://docs.github.com/en/communities/moderating-comments-and-conversations/locking-conversations)
+  (requires Maintainer role or [Organization Moderator role](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-moderators-in-your-organization))
+
```

Comment:
I'd suggest to add a corresponding section also for Google (Groups), and perhaps GitLab. 
(Didn't get to that when I was working on this section in #36844.)

Reply:
Good idea, and I forgot to point out in the description that I had taken this from your changes in #36844.

Reply:
Done for Google Groups. I don't know how widely GitLab is used, so I changed the heading to something about "common project platforms," to try to avoid the implication that this list is complete.

Reply:
Regarding GitLab, see the end of the org README at https://github.com/sagemath/ -- it's only in use for hosting some small adopted projects. 
I think it's fine as is.